phpBB/styles/prosilver/theme/content.css

```diff
@@ -517,10 +516,9 @@ blockquote .codebox {
 	display: block;
 	height: auto;
 	max-height: 200px;
-	padding-top: 5px;
+	padding: 8px 3px 5px 3px;
```

Comment:
Sure but inside the actual coebox, there appears to be 3px more padding above the code content than at the bottom of the content because of this being 8px.